bot/seasons/evergreen/nationday.py

```diff
@@ -0,0 +1,149 @@
+import datetime
+import json
+import logging
+from pathlib import Path
+from typing import List, Tuple
+
+import discord
+import requests
+from discord.ext.commands import Bot, Cog, Context, command
+
+from bot.pagination import ImagePaginator
+
+
+logger = logging.getLogger(__name__)
+# RESTful API to get countries info
+url = "https://restcountries.eu/rest/v2/alpha/"
+# white flag image to show during error
+white_flag = "https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/\
+White_flag_waving.svg/158px-White_flag_waving.svg.png"
```

Comment:
To multi-line a string literal, it's nicer to do:
```python
white_flag = (
    "abcd"
    "efgh"
)
```
Notice no separating comma.